plugins/CoreAdminHome/SystemSettings.php

```diff
@@ -63,9 +73,47 @@ private function createTrustedHostnames()
         });
     }
 
+    private function createDisabledDimensions()
+    {
+        return $this->makeSetting('disabled_settings', $default = [], FieldConfig::TYPE_ARRAY, function (FieldConfig $field) {
+            $field->introduction = 'Disabled Dimensions'; // TODO translate
+            $field->inlineHelp = "Disable dimensions to avoid tracking this data no matter what is sent to the tracker. This can be useful in being compliant with various privacy regulations."; // TODO: translate
+            $field->uiControl = FieldConfig::UI_CONTROL_MULTI_SELECT;
+            $field->availableValues = $this->getAvailableDimensionsToDisable();
+        });
+    }
+
     public function save()
     {
         parent::save();
         Cache::deleteTrackerCache();
     }
+
+    private function getAvailableDimensionsToDisable()
+    {
+        // TODO: translate types
+        $dimensions = [];
+        $this->addDimensions($dimensions, VisitDimension::getAllDimensions(), $type = 'Visit');
+        $this->addDimensions($dimensions, ActionDimension::getAllDimensions(), $type = 'Action');
```

Comment:
I reckon for now we can probably remove at least `Action` dimensions from this list. I think we could maybe in the beginning even remove conversion dimensions as they mostly get the value from a `visit` dimension anyway. So we'd only allow to disable visit dimensions for now